packages/experiments/src/components/Chiclet/examples/Chiclet.Xsmall.Example.scss

```diff
@@ -0,0 +1,67 @@
+.footer {
+  height: 36px;
+  padding-left: 0px;
+}
+
+.activities {
```

Comment:
nit: not a blocker -> can you please add all this styles using merge-styles inside the example file itself. We try to remove all .scss files for our examples.